### ISSUE TITLE
to_csv does not support DataFrame indexes

### ISSUE THREAD

**Issue by kbzowski** [opensource9ja/danfojs#282]
Indexes in DataFrame are not saved when exporting to csv using to_csv.

`  const df = new DataFrame(input, { index: ['A', 'B', 'C'] } );
  df.to_csv(filename);`

**Comment by Emporea** [comment]
Is this related to #361?
toJSON() also didnt't include the index.

**Comment by risenW** [comment]
> Is this related to #361? toJSON() also didnt't include the index.
> 
> It seems like most of the io files need an update so that Series and DataFrames include the index too.

Yes, it's similar. If you want and can, you can raise a separate PR for this.
